src/VirtoCommerce.InventoryModule.Core/ModuleConstants.cs

```diff
@@ -33,20 +34,27 @@ public static class General
                     DefaultValue = "50",
                 };
 
-                public static IEnumerable<SettingDescriptor> AllSettings
+                public static IEnumerable<SettingDescriptor> AllSettings => new[] { PageSize };
+            }
+
+            public static class Search
+            {
+                public static SettingDescriptor EventBasedIndexationEnable = new SettingDescriptor
                 {
-                    get
-                    {
-                        yield return PageSize;
-                    }
-                }
+                    Name = "Inventory.Search.EventBasedIndexation.Enable",
+                    GroupName = "Search | Inverntory",
```

Comment:
Just lapsus: "Inventory"

Reply:
Why is that? The module is done as well.

Reply:
Done